deployment/cdk/opensearch-service-migration/lib/migration-assistance-stack.ts

```diff
@@ -41,6 +42,45 @@ export class MigrationAssistanceStack extends Stack {
         }
     }
 
+    validateAndReturnVPCSubnets(vpc: IVpc, brokerNodeCount: number, azCount: number, specifiedSubnetIds?: string[]): string[] {
```

Comment:
Please add something about MSK in the function name since this is specific to MSK (and not another service of ECS/externally-managed kafka)